modules/ocf_admin/manifests/create/redis.pp

```diff
@@ -40,14 +38,21 @@
     notify  => Service['redis-server'];
   }
 
-  ocf::repackage { 'stunnel4': backport_on => jessie } ->
-  augeas { '/etc/default/stunnel4':
-    lens    => 'Shellvars.lns',
-    incl    => '/etc/default/stunnel4',
-    changes =>  ['set ENABLED 1'],
-  } ~>
-  file { '/etc/stunnel/redis.conf':
-    content => template('ocf_admin/stunnel/redis.conf.erb'),
-  } ~>
-  service { 'stunnel4': }
+  package { 'hitch': }
```

Comment:
I think just putting this in the package array at the top of the file would be easier and a bit cleaner, but keeping it here does make some sense since it's grouped with the other hitch stuff then. I'd say it's up to you either way, but personally I'd put it at the top of the file with the `redis-server` package.

Reply:
Also a good suggestion; that's a leftover from the `->` chaining in the original code.